portal-web/docroot/html/taglib/ui/search_iterator/list.jsp

```diff
@@ -274,9 +274,17 @@ if (fixedHeader) {
 					if (data == null) {
 						data = new HashMap<String, Object>();
 					}
+
+					JSONObject rowJSONObject = JSONFactoryUtil.createJSONObject(
+						row.getData(
+						).get(
+							"value"
+						).toString());
+
+					String rowName = rowJSONObject.getString("name");
 				%>
 
-					<tr class="<%= GetterUtil.getString(row.getClassName()) %> <%= row.getCssClass() %> <%= row.getState() %> <%= rowIsChecked ? "active" : StringPool.BLANK %>" data-qa-id="row" id="<%= rowElementId %>" <%= AUIUtil.buildData(data) %>>
+					<tr aria-label="<%= LanguageUtil.format(request, searchContainerRowAriaLabel, rowName) %>" class="<%= GetterUtil.getString(row.getClassName()) %> <%= row.getCssClass() %> <%= row.getState() %> <%= rowIsChecked ? "active" : StringPool.BLANK %>" data-qa-id="row" id="<%= rowElementId %>" <%= AUIUtil.buildData(data) %> <%= !Validator.isBlank(searchContainerRowAriaLabel) ? "tabindex=\"0\"" : StringPool.BLANK %>>
```

Comment:
This is still too much being added to search iterator. We can't assume that name is available and we shouldn't be calling LanguageUtil.format here. We should be passing in a translated string. Also if we need tabindex we should add that as another property of searchContainerRow instead of trying to determine when it should be added.

Reply:
@ethib137 I wanted to pass in a translated string, but I don't have access to each entry name/title in`item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp` in the tag `liferay-ui:search-container-row`. 

I can add `tabindex` as another attribute.

Reply:
@pat270 Yep... Since we got the approval from Marcos, let's pass a simple translated string: "Select the item and close the modal." We should also pass the tabindex parameter and not assume we know when to add it. Finally, we need to only add these properties when the item selector is single selection, NOT multiselect. Let me know if you have any questions. Thanks.

Reply:
Thanks for catching the multiselect @ethib137 . I added the first two. I still need to exclude multiselect.